### ISSUE TITLE
prepare the call for participation

### ISSUE THREAD

**Issue by aoloe** [libregraphicsmeeting/htdocs-2017#2]
- [x] create the form with wp libre form
- [ ] make sure that all fields are needed (and no fields are missing)
  - [ ] does it make sense to ask for all those travel costs reimbursement infos?
  - [x] would it make more sense to eventually ask for an approximation in us dollars / eur /  reals (pick one of them)
- [ ] eventually, send a confirmation email to the submitter (is it really needed?)
- [ ] program a way to export all entries to csv
   - [ ] check https://github.com/k1sul1/wplf-export/
- [ ] ...
- [ ] port the php script for formidable to show the entries in schedule

**Comment by jsbueno** [comment]
Yes -  I thnk the trave l cost could be simplified - just the a total travel cost estimate - in USD, EUR or BRL - and remove the limit on money/Km information (0,10$).

The e-mail for the submiter is not needed if it is hard to do  - but if it already works, it could be kept.  

You talk about generating an CSV - how are the entries kept? At a database running there?

**Comment by aoloe** [comment]
- are you really going to try to reimburse the traveling costs?
- i would have to implement the email to the submitter (i will have to dive into it, but i'm really not sure that i can do it before friday)
- there is a list of entries in the form management. (there are two of them in the trash, which you can re-enable)

**Comment by aoloe** [comment]
ok, i've made the travelling costs part shorter...

**Comment by jsbueno** [comment]
As for the travel costs:  realistically - I think people who really need it can get it some months later, but since we did not improve our mechanisms, only usng Louis's Canadian organization  - we know it takes time, but it can work. 

So, maybe the text could be changed to give emphasis on that - ah, I found out where to edit it - I think it is alright now. 

Forget about the e-mail if programming is needed.

I think we are set for publishing it now - and them advertising it on the mailing lists.
